portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -62,13 +63,14 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
       if(maxComments <= i){
         break;
       }
+      String key = KeyFactory.keyToString(entity.getKey());
```

Comment:
Do you need to convert it to string? Can we just store key in its original type?

Reply:
I tried, but when I wanted to send it as a parameter in the post request I couldn't send it as a key, so I changed it to String, but I think I could store both.

Reply:
Ah you're right, yeah in that case just store as a string.